<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>SelectAlgorithmWindow</class>
  <widget class="QDialog" name="SelectAlgorithmWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>250</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Select Matching Algorithm</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="title_label">
          <property name="text">
            <string>Select Matching Algorithm</string>
          </property>
          <property name="font">
            <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
            </font>
          </property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item row="1" column="0">
        <widget class="QFrame" name="algorithm_frame">
          <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
              <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                  <widget class="QLabel" name="algorithm_label">
                    <property name="text">
                      <string>Algorithm</string>
                    </property>
                    <property name="font">
                      <font>
                        <pointsize>12</pointsize>
                      </font>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QComboBox" name="algorithm_combo">
                    <item>
                      <property name="text">
                        <string>Reciprocal Best Matching</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Greedy Matching</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Hungarian Matching</string>
                      </property>
                    </item>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item row="1" column="1">
        <widget class="QFrame" name="models_frame">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                  <widget class="QLabel" name="models_label">
                    <property name="text">
                      <string>Models</string>
                    </property>
                    <property name="font">
                      <font>
                        <pointsize>12</pointsize>
                      </font>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QComboBox" name="models_combo">
                    <item>
                      <property name="text">
                        <string>All</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Only Unmatched</string>
                      </property>
                    </item>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
<!--
      <item row="1" column="2">
        <widget class="QFrame" name="filter_frame">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
              <widget class="QPushButton" name="filter_button">
                <property name="text">
                  <string>Filter Model</string>
                </property>
                <property name="font">
                  <font>
                    <pointsize>12</pointsize>
                  </font>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>120</width>
                    <height>50</height>
                  </size>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
-->
      <item row="2" column="1">
        <widget class="QPushButton" name="load_button">
          <property name="text">
            <string>Load</string>
          </property>
          <property name="font">
            <font>
              <pointsize>12</pointsize>
            </font>
          </property>
          <property name="minimumSize">
            <size>
              <width>100</width>
              <height>40</height>
            </size>
          </property>
        </widget>
      </item>
      <item row="2" column="2">
        <widget class="QPushButton" name="back_button">
          <property name="text">
            <string>Back</string>
          </property>
          <property name="font">
            <font>
              <pointsize>12</pointsize>
            </font>
          </property>
          <property name="minimumSize">
            <size>
              <width>100</width>
              <height>40</height>
            </size>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
